sklearn/semi_supervised/_label_propagation.py

```diff
@@ -176,11 +176,11 @@ def predict_proba(self, X):
 
         Parameters
         ----------
-        X : array-like of shape (n_samples, n_features)
+        X : ndarray of shape (n_samples, n_features)
 
         Returns
         -------
-        probabilities : array, shape = [n_samples, n_classes]
+        probabilities : ndarray of shape = [n_samples, n_classes]
```

Comment:
`of shape (n_samples, n_classes)`

Reply:
same in other places: use parenthesis

Reply:
Thanks @NicolasHug  have made the edits.